trigger/build.go

```diff
@@ -285,6 +285,20 @@ func (t *buildTriggers) buildAttachments(data *commonTemplateData) []message.Sla
 		Title:     fmt.Sprintf("Build: %s", t.build.DisplayName),
 		TitleLink: data.URL,
 		Text:      taskStatusToDesc(t.build),
+		Fields: []*message.SlackAttachmentField{
+			{
+				Title: "Version",
+				Value: fmt.Sprintf("<%s|%s>", t.build.Version, versionLink(t.uiConfig.Url, t.build.Version)),
+			},
+			{
+				Title: "Makespan",
+				Value: t.build.ActualMakespan.String(),
+			},
+			{
+				Title: "Time Taken",
```

Comment:
Should this one be Duration as well so that people know it's the same as the `"Duration"` measure on the other slack notifications?